tests/test_routers/test_utterances.py

```diff
@@ -74,6 +74,16 @@ def test_get_utterances(app: FastAPI, monkeypatch):
     assert len(first_utterance["modelSaliency"]) == 2
 
 
+def test_get_utterances_empty_filters(app: FastAPI):
+    client = TestClient(app)
+    resp = client.get("/dataset_splits/eval/utterances?utterance=yukongold").json()
+    assert len(resp["utterances"]) == 0
+
+# This used to fail when we were filtering on an empty dataset
```

Comment:
I think black will tell you to
```suggestion
    # This used to fail when we were filtering on an empty dataset
```